Directory.Build.props

```diff
@@ -220,7 +220,7 @@
     <MvcTestingTargets>$(MSBuildThisFileDirectory)src\Mvc\Mvc.Testing\src\Microsoft.AspNetCore.Mvc.Testing.targets</MvcTestingTargets>
     <_MvcTestingTasksAssembly>$(ArtifactsBinDir)\Microsoft.AspNetCore.Mvc.Testing.Tasks\$(Configuration)\netstandard2.0\Microsoft.AspNetCore.Mvc.Testing.Tasks.dll</_MvcTestingTasksAssembly>
     <!-- IIS native projects can only be built on Windows for x86/x64/ARM64. -->
-    <BuildIisNativeProjects Condition=" '$(TargetOsName)' == 'win' AND ('$(TargetArchitecture)' == 'x86' OR '$(TargetArchitecture)' == 'x64' OR '$(TargetArchitecture)' == 'ARM64') ">true</BuildIisNativeProjects>
+    <UpdateIisNativeAssets Condition=" '$(TargetOsName)' == 'win' AND ('$(TargetArchitecture)' == 'x86' OR '$(TargetArchitecture)' == 'x64' OR '$(TargetArchitecture)' == 'ARM64') ">true</UpdateIisNativeAssets>
```

Comment:
IIS native projects can only be built on Windows, but also they are always built on Windows. When can this property be false on Windows? I currently set it when I build Kestrel for instance, even from a fresh clone. Can some project build.cmd files set it to `false` safely?

Reply:
I believe you can just pass in -noBuildNative if you explicitly want to skip native code building as that turns off.  But hence the rename here, there's a bunch of projects that need to update iis native assets which is why we are renaming this to something not called build

Reply:
Basically projects that want to pull in ancm will have this set, and if you don't have ancm built, when it tries to update/include it, it will fail, as it should

Reply:
`-noBuildNative` won't change most of what the property does. That switch only controls whether a native (desktop) `msbuild` happens if you use one of our build.cmd or build.ps1 scripts.
The `$(BuildNative)` property reflect that choice while we're building but only when building using desktop `msbuild` and mainly to distinguish the scenario from `-buildInstallers` / `$(BuildInstallers)` (the other case where we build using desktop `msbuild`).